notebooks/Tutorial_Top-K_Multidimensional_Motif_and_Matches_Discovery.ipynb

```diff
@@ -0,0 +1,827 @@
+{
```

Comment:
<p>I don't think it's entirely clear why we get three appliances. At least, it's not clear to me :)</p><p> </p><p>Also, what is the difference between "Fridge-Freezer" and "Freezer"? Perhaps the latter is what American's might refer to as a "Chest Freezer"? We might want to clarify it here if we can.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/557/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>